src/CoreFx.Private.TestUtilities/src/System/PlatformApis.Unix.cs

```diff
@@ -203,12 +203,13 @@ private static DistroInfo LoadDistroInfo()
                     }
                 }
             }
+            // possibly should fall back to /usr/lib/os-release here
```

Comment:
What is this comment referring to?  A future todo?  The code that follows?  What's the action item here?

Reply:
As noted in the linked issue, the man pages mention this file may be used as a fallback in the absence of etc/os-release. I do not think we need to do anything unless this emerges as a problem.